charts/azuremonitor-containers/Chart.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: v1
 appVersion: 7.0.0-1
 description: Helm chart for deploying Azure Monitor container monitoring agent in Kubernetes
 name: azuremonitor-containers
-version: 2.9.4
+version: 2.9.5
```

Comment:
Let bump the chart major version for this  to cover both non-AKS & Azure Arc K8s extension scenarios.